test/conftest.py

```diff
@@ -44,7 +42,10 @@ def load_yaml(fpath):
     # Parse the yaml
     obj = oyaml.safe_load(raw)
     # Return the parsed and structured object
-    return process_struct(obj)[0]
+    if process_struct(obj):
```

Comment:
Nitpick: Is `process_struct()` potentially an expensive call? I'm wondering if it may be worthwhile to store the result in a variable to avoid calling it twice here.